src/option_parser.cr

```diff
@@ -316,13 +316,26 @@ class OptionParser
     @flags.join io, '\n'
   end
 
+  # Width for option list portion of summary.
+  property summary_width : Int32 = 32
+
+  def summary_width=(width : Int32)
+    raise ArgumentError.new("Negative summary width: #{width}") if width < 0
+    @summary_width = width
+  end
+
+  # Indentation for summary.
+  property summary_indent : String = "    "
+
   private def append_flag(flag, description)
-    indent = " " * 37
-    description = description.gsub("\n", "\n#{indent}")
-    if flag.size >= 33
-      @flags << "    #{flag}\n#{indent}#{description}"
+    width = summary_width.clamp(0..)
```

Comment:
Redundant `clamp` call.

Reply:
I simply deleted the call.
https://github.com/crystal-lang/crystal/pull/15326/commits/48a0ef4681f7ac3bcefd4e3a304d01859f1cc795